lib/src/main/java/com/orange/ods/compose/component/bottomnavigation/OdsBottomNavigation.kt

```diff
@@ -99,25 +118,60 @@ object OdsBottomNavigation {
 
         @Composable
         override fun RowScope.Content(modifier: Modifier) {
-            BottomNavigationItem(
-                selected = selected,
-                onClick = onClick,
-                icon = { icon.Content() },
-                enabled = enabled,
-                label = label?.let {
-                    {
-                        Text(
-                            text = label,
-                            maxLines = 1,
-                            overflow = TextOverflow.Ellipsis,
-                            style = OdsTheme.typography.caption
-                        )
-                    }
-                },
-                alwaysShowLabel = alwaysShowLabel,
-                selectedContentColor = OdsTheme.colors.component.bottomNavigation.itemSelected,
-                unselectedContentColor = OdsTheme.colors.component.bottomNavigation.itemUnselected
-            )
+            val itemPadding = dimensionResource(id = R.dimen.spacing_s)
```

Comment:
This variable is only used for the selected line, thus maybe `selectedLinePadding` or `selectedLineMargin` (`margin` is the parameter name of the `linkTo` method) would be more appropriate?